toolbox/CMakeLists.txt

```diff
@@ -244,6 +253,10 @@ else()
     target_link_libraries(WBToolbox ${LINKED_LIBRARIES} ${Matlab_LIBRARIES})
 endif()
 
+# Remote Inverse Kinematics requires C++11
+set(CXX_STANDARD_REQUIRED ON)
+set_property(TARGET WBToolbox PROPERTY CXX_STANDARD 11)
```

Comment:
Ok.
Consider that I would have liked to use directly https://cmake.org/cmake/help/v3.1/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html but unfortunately I couldn't find the `mutex` feature